librz/analysis/dwarf_process.c

```diff
@@ -1,6 +1,7 @@
 // SPDX-FileCopyrightText: 2012-2020 houndthe <cgkajm@gmail.com>
 // SPDX-License-Identifier: LGPL-3.0-only
 
+#include "rz_util/rz_strbuf.h"
```

Comment:
I think strbuf is the part of `rz_util.h`

Reply:
also, use `<rz_util/rz_strbuf.h>`